### PR TITLE
Fix 1441 minimum needs result

### PR DESCRIPTION
**Pull request by ismailsunni**
For #1441

### CODE REVIEW

**Comment by Charlotte-Morgan** [comment]
hi @ismailsunni - this is looking good.
- [ ] I have looked at flood raster on population raster with aggegation (jakarta) and its fine. 
so in this case affected = evacuated = basis for minimum needs. I'd like to see rounding in the general & detailed reports. But that can wait for IF review. we probably need something that states that all affected people are assumed to be evacuated etc
- [ ] i have looked at flood polygon on population raster with aggregation (jakarta). In this case evacuated = 1% of affected and so the numbers in the reports are different. I like the extra information you include in the detailed minimum needs report (for people needing evacuation). Things get messy though with the detailed gender report and the detailed age report. Both are for affected and not the evacuated population; but then hygiene packs and additional rice are calculated on the affected rather than evacuated population. It would be helpful to add (affected people) to the title for the detailed gender and detailed age reports. Please add this to things to be included in the IF review
# cc
@adissadis

**Comment by ismailsunni** [comment]
For point 1, basically we want to do it in IF review.
I will put in the notes that all affected people are assumed to be evacuated for now. So, for flood raster on population raster, it will give report like:
selection_037


For point 2:
For flood polygon and population raster.
I have added extra notes in the table header. See below:
selection_038

Is that ok?

Actually, I added the same table header for population-IF, for the detailed table. To make them uniform.

File: safe/impact_statistics/aggregator.py [hunk 1]
```diff
@@ -949,27 +946,27 @@ def _aggregate_line_impact(self, safe_impact_layer):
             tmp_filename = unique_filename(
                 suffix='.shp', dir=output_directory)
             res = self.processing.runalg('qgis:exportaddgeometrycolumns',
-                projected_layer,
-                # 2, # Ellipsoidal
-                0,  # Layer CRS
-                tmp_filename)
+                                         projected_layer,
```
Comment (timlinux) [inline safe/impact_statistics/aggregator.py]: it was correctly indented above (pull left!)

**Comment by timlinux** [comment]
Great that is nice and clear thanks! LGTM

**Pull request merged**